components/IntakeForm/index.js

```diff
@@ -0,0 +1,74 @@
+import { useState } from "react";
+import { TrackerForm } from "./IntakeForm.styled";
+import { SubmitButton } from "../MachineForm/MachineForm.Styled";
+import { FormInput } from "../MachineForm/MachineForm.Styled";
+
+export default function IntakeForm({ onsubmit }) {
+  const [carbs, setCarbs] = useState("");
+  const [protein, setProtein] = useState("");
+  const [fat, setFat] = useState("");
+  const [kcal, setKcal] = useState("");
+
+  const handleCarbsChange = (event) => {
+    const value = event.target.value;
+    setCarbs(value);
+    calculateKcal(value, protein, fat);
+  };
+
+  const handleProteinChange = (event) => {
+    const value = event.target.value;
+    setProtein(value);
+    calculateKcal(carbs, value, fat);
+  };
+
+  const handleFatChange = (event) => {
+    const value = event.target.value;
+    setFat(value);
+    calculateKcal(carbs, protein, value);
+  };
+  const calculateKcal = (carbs, protein, fat) => {
+    const calculatedKcal = carbs * 4 + protein * 4 + fat * 9;
+    setKcal(calculatedKcal);
+  };
+
+  return (
+    <>
```

Comment:
We don't need the React Fragment in this case, as we are only returning one element from `IntakeForm`.  (i.e. `TrackerForm`)